subsys/bluetooth/controller/bt_ll_softdevice_hci_rpmsg.conf

```diff
@@ -1,6 +1,5 @@
 # Configuration file passed as an overlay when building
 # zephyr/samples/bluetooth/hci_rpmsg as a child image.
 
-CONFIG_BT_LL_SOFTDEVICE_DEFAULT=y
-CONFIG_BT_MAX_CONN=4
```

Comment:
That would increase the BT_MAX_CONN to 16 (default value from Kconfig in Zephyr). Do we need this ?

Reply:
I don't think so, but the default was 16. This config file wasn't be used previously. Since the Network core runs only hci_rpmsg and has memory resources I don't think we need to limit this

Reply:
@KAGA164 We can use as much memory as required, but please revert the changes to the maximum data length of the controller.